t4_devkit/schema/tables/object_ann.py

```diff
@@ -77,9 +77,10 @@ class ObjectAnn(SchemaBase):
         validator=validators.deep_iterable(validators.instance_of(str))
     )
     bbox: RoiLike = field(converter=tuple, validator=is_roi)
-    mask: RLEMask = field(
+    mask: Optional[RLEMask] = field(
```

Comment:
[WANT] Let's use shorthand expression.

```suggestion
    mask: RLEMask | None = field(
```